extensions/wikia/LocalNavigation/styles/LocalNavigation.scss

```diff
@@ -29,7 +30,9 @@ $local-nav-wordmark-right-spacing-small: 30px;
 	@include font-size-xxs;
 	border-bottom: 1px solid $color-page-border-subtle;
 	height: 80px;
-	padding: 0 $medium-content-padding + $medium-grid-column-padding - $content-border-width;
+	@if $widthType != 0 {
```

Comment:
It's a leftover from this: https://github.com/Wikia/app/blob/dev/skins/oasis/css/wikiagrid.scss#L6 As it's only temporary solution (and it will be removed in https://wikia-inc.atlassian.net/browse/CONCF-224) and it's easier-to-grep right now, I would like to leave this as-is.